__tests__/label.test.js

```diff
@@ -0,0 +1,103 @@
+import { random } from 'faker';
+import { launchApp, shutdownApp, clearDatabaseState } from './helpers.js';
+
+describe('Label', () => {
+  let app;
+
+  beforeAll(async () => {
+    app = await launchApp();
+  });
+
+  afterAll(async () => {
+    await shutdownApp(app);
+  });
+
+  beforeEach(async () => {
+    await clearDatabaseState(app);
+  });
+
+  describe('create', () => {
+    it('should return 302 when using valid name', async () => {
```

Comment:
As we are writing expectations here, I'd also add something like "and create label"

Reply:
done